routers/repo/issue.go

```diff
@@ -1014,6 +1014,8 @@ func UpdateIssueStatus(ctx *context.Context) {
 			ctx.ServerError("ChangeStatus", err)
 			return
 		}
+
+		notification.NotifyIssueChangeStatus(ctx.User, issue, isClosed)
```

Comment:
Only send notifications if status really change. In my understanding this will send a notification if `isClosed = true` and the issue is already closed.